### PR TITLE
Contact form feature1

### PR DESCRIPTION
**Pull request by Tshepo45**
-Modified .html and .CSS files
-Coding partners contributed commits to contact form

### CODE REVIEW

**Review by osamaashraf6 — COMMENTED** [review]
Hi @Tshepo45 😊, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🆘💀

- You have submitted a project, that has no information for your partner in the Readme, so please try to add the info for contact for your partner in the Readme 
- Kindly try to add the title of the project in the Readme up the description of the Project to make the readme professional 😃

| Your readme  | Your readme |
| --- | --- |
|   image   |   image    |





Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,232 +1,255 @@
...
+         </ul>
+         <div>
+          <input id="get-submit"  type="submit" value="Get It Touch" class="btn-n"/>
+          <img src="image/Separator2.png" class="shape"  alt="shape">
+        </div>
+         
+        
+      </form>
+    </div>
+  </section>
```
Comment (bobb-Rob) [inline index.html:253]: Well done on this project.

I notice a few things that I believe will improve your project even better. Ple note as follows 👇🏾 
- There should be a background illustration in this section.
- The space between the input elements is too large and should follow the size as seen in the Figma template.
- The `get in touch` button does not match the button in the Figma template.
- The paragraph text below the `contact me` title is not well aligned in comparison with the Figma design.


Your projects | Figma Design
---- | ----
image | image 
 
 image

**Review by bobb-Rob — APPROVED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Tshepo45, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (bobb-Rob) [reply index.html:253]: Please treat the above review comments as changes required as it was mistakenly selected as **approved**.

**Review by TadesseDev — APPROVED** [review]
### Hi @Tshepo45 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**